api/handler/api.go

```diff
@@ -16,14 +16,14 @@ type (
 		cfg *Config
 	}
 
-	// Config contains data which handler need to keep.
+	// Config contains data which handler needs to keep.
 	Config struct {
 		DefaultPolicy *netmap.PlacementPolicy
 		DefaultMaxAge int
 	}
 )
 
-// DefaultPolicy is a default policy of placing container in NeoFS if it's not set at the request.
+// DefaultPolicy is a default policy of placing containers in NeoFS if it's not set otherwise at the request.
```

Comment:
Is `otherwise` suitable there? Looks a bit odd to me for some reason.

The context is that `Policy` may be omitted in the request. If so, then use `DefaultPolicy`.

Reply:
You are right, it's unnecessary here. For some reason I've put `otherwise` to emphasize that something is set not like in the default.